src/test/java/org/ohdsi/authenticator/service/rest/RestTemplateProviderTest.java

```diff
@@ -0,0 +1,58 @@
+package org.ohdsi.authenticator.service.rest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.ohdsi.authenticator.service.rest.config.ProxyConfig;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+public class RestTemplateProviderTest {
+
+    public static final String TEST_URL = "https://www.arachnenetwork.com/api/v1/build-number";
+    //this proxy ip is not fixed, so it could change. To fix this use any available proxy
+    public static final String PROXY_URL = "167.99.232.18";
```

Comment:
Where is this magic IP from? Cannot we avoid having new tests requiring an internet connection to complete?

Reply:
good point, I don't like this either. This is a random proxy that I have found on the internet. I plan to replace this one with our own proxy that Konstantin is working on. There is an option to set up an embedded proxy for all unit tests(timewise, I am not sure that it worth it ). and the last option is just to get rid of the test.

Reply:
> I plan to replace this one with our own proxy that Konstantin is working on

That doesn't solve the conceptual issue

> There is an option to set up an embedded proxy for all unit tests

Yes, that's what we should go for

Reply:
fixed